### ISSUE TITLE
FormatException: Unexpected character

### ISSUE THREAD

**Issue by VictorPadovan1997** [CodesFirst/new_version_plus#3]
Hello guys, I have this error occurring in me application.
Do you have one solution for my problem? 

This my code pass my packageName
```
 final newVersion = NewVersionPlus(
        iOSId: 'package_name',
        androidId: 'package_name',
      );
      final status = await newVersion.getVersionStatus();
      print(status);
```

And this Exception occurring when method final status = await newVersion.getVersionStatus(); is activated.
```
Exception has occurred.
FormatException (FormatException: Unexpected character (at character 5366)
...end email to guests without owners" knowledge"]],["event"]],["Phone",[nu...
{"key": "ds:5", "hash": "9", "data":[[[[]]],[null,null,[["App_name"],null,null,null,null,null,null,null,[false,true,[false]],["All ages",[null,2,[512,512],[null,null,"https://play-lh.googleusercontent.com/_KiRouu_G6J_2jwePzQ_i5_FMc_SVKT3mI7d7KKq9zca-Nr8bj2bPasawLvk6ajzASQS-90a8hYXeAh0lQ"]],null,null,[null,"\u003ca href\u003d\"https://support.google.com/googleplay?
                                ^
);
```
Remember : my application is in a development environment, we don't go up to production in stores(apple/ play), we are passing existing package for testing.
Thanks for me help.

**Comment by j-j-gajjar** [comment]
+1

**Comment by j-j-gajjar** [comment]
@VictorPadovan1997  
Have you used any special character in the name of your application?

**Comment by MhdYhyaAlshaki** [comment]
+1

**Comment by pewiher** [comment]
Hello how are you, an update was made, now the check is done with regular expressions, could you validate if your problem was resolved, the version is 0.0.4

**Comment by pewiher** [comment]
We are waiting for your comments, if you do not respond within 2 days, the case will be considered closed.
